src/View/Helper/UrlHelper.php

```diff
@@ -48,6 +48,9 @@ public function build($url = null, $options = false)
             'escape' => true,
         ];
         if (!is_array($options)) {
+            deprecationWarning(
+                'The `bool` array option is deprecated. Use an array here instead with `\'fullBase\' => true`.'
+            );
```

Comment:
This one causes a ton of failures in the test suite, and I'm not sure it is a high value change to be making this late in 3.x's life. It also doesn't add a significant burden to us to maintain going forwards.

Reply:
You mean we shouldnt throw the deprecation warning?

Reply:
I added it to shim plugin instead: https://github.com/dereuromark/cakephp-shim/pull/53

Reply:
Yes, if this deprecation causes many of our tests to fail it will likely be tedious for application developers too.

Reply:
I'm :+1:  with dont adding more deprecation warning to cake 3.8
we can add it since cake 4.0